### DISCUSSION TITLE
initial_prompt Not working after 30 sec

### DISCUSSION THREAD

**Discussion by Abdurrafey-Siddiqui** [openai/whisper#344] [Q&A] [answered]
initial_prompt only seems to work for the first 30 seconds of the audio since the rest of my audio seems to not be using it. how to fix??

**Comment by jianfch** [comment, marked as answer]
It might be due to one of conditions being true
https://github.com/openai/whisper/blob/d18e9ea5dd2ca57c697e8e55f9e654f06ede25d0/whisper/transcribe.py#L238
if you didn't set `condition_on_previous_text=False` and temperature of the result > 0.5, then specifying temperature(s) less than 0.5 will fix it such as `temperature=0.0` or `temperature=(0.0, 0.2, 0.4)`
